packages/stitch/src/getFieldsNotInSubschema.ts

```diff
@@ -175,7 +176,7 @@ export function extractUnavailableFieldsFromSelectionSet(
           });
         }
       } else {
-        unavailableSelections.push(selection);
+        // unavailableSelections.push(selection);
```

Comment:
What happens here, why is this part commented out?

Reply:
If unnecessary/unreachable, rather remove.

Reply:
Good catch. It should be removed already.